packages/paste-website/src/pages/introduction/for-designers/design-guidelines/index.mdx

```diff
@@ -60,53 +56,136 @@ account through [ServiceNow](https://twilio.service-now.com/sp/). When you have
+
+#### Paste libraries
+
+<UnorderedList>
+  <ListItem>
+    <Box display="inline">
+      <Box display="inline-block" verticalAlign="middle">
+        <Badge as="span" variant="new">
+          <NewIcon decorative />
+          New
+        </Badge>
+      </Box>
+      {` `}
+      <Anchor
+        href="https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/%5BWIP%5D-Paste-Components?node-id=3015%3A46646&t=2Fe524au9O9rlWZU-1"
+        showExternal
+      >
+        Paste Components
+      </Anchor>
+      <Text display="inline">
+        {` `} includes the latest updated components that use the updated design language themes. All new components and
+        component updates will be made in this library.
+      </Text>
+    </Box>
+  </ListItem>
+  <ListItem>
+    <Box display="inline">
+      <Box display="inline-block" verticalAlign="middle">
+        <Badge as="span" variant="new">
+          <NewIcon decorative />
+          New
+        </Badge>
+      </Box>
+      {` `}
+      <Anchor
+        href="https://www.figma.com/file/U0oTLE09SGK7svnr818HYT/%5BWIP%5D-Paste-Twilio-Theme?node-id=3558%3A32533&t=PYbe5AovStxUbEjF-1"
+        showExternal
+      >
+        Twilio Paste Theme
+      </Anchor>
+      <Text display="inline">
+        {` `} includes the newest design language token updates (color, text, and shadow styles), needed for the Paste
+        Components library.
+      </Text>
+    </Box>
+  </ListItem>
+  <ListItem>
+    <Box display="inline">
+      <Box display="inline-block" verticalAlign="middle">
+        <Badge as="span" variant="new">
+          <NewIcon decorative />
+          New
+        </Badge>
+      </Box>
+      {` `}
+      <Anchor
+        href="https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/%5BWIP%5D-Paste-Components?node-id=3015%3A46646&t=2Fe524au9O9rlWZU-1"
+        showExternal
+      >
+        Twilio Paste Dark Theme
+      </Anchor>
+      <Text display="inline">{` `}includes the newest token updates for the dark theme.</Text>
+    </Box>
+  </ListItem>
+  <ListItem>
+    <Anchor href="https://www.figma.com/file/S4z0Kqjb9AYosnkRQldLte/Paste-Patterns" showExternal>
+      Paste Patterns
+    </Anchor>
+  </ListItem>
+  <ListItem>
+    <Anchor href="https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations" showExternal>
+      Brand Illustrations
+    </Anchor>
+    {` `}includes illustrations for empty states, sections, etc
+  </ListItem>
+  <ListItem>
+    <Anchor href="https://www.figma.com/file/GfY11A2VW638eFK48nDtdE/Asset-Documentation" showExternal>
+      Asset Documentation
+    </Anchor>
+    {` `}includes helpful utilities for marking up designs, like sticky notes and cursors
+  </ListItem>
+  <ListItem>
+    <Anchor href="https://www.figma.com/file/RfZYRjUH56rZ0qe0veqRJA/File-and-Cover-Sheet-Template" showExternal>
+      File and Cover Sheet Template
+    </Anchor>
+  </ListItem>
+</UnorderedList>
+
+<Callout variant="neutral">
```

Comment:
Question for @serifluous : I'm thinking we could remove this callout, since we're not referencing the old libraries on this page. Can you let me know what you think?

Reply:
(see slack)